snprc_ehr/resources/queries/study/DemographicsHL7Deleted.sql

```diff
@@ -0,0 +1,3 @@
+select
+       '0186B9BD-E5EF-406A-B67B-2BD902A0BE4C' as objectId,
+       now() as modified
```

Comment:
Is this file needed?

Reply:
Good catch. I removed it from the PR - I created it when I was troubleshooting the missing schema issue. As for the binary SQL files, they were created in SSMS and saved with UTF-16LE encoding (Microsoft's default) - I converted them to UTF-8.